sequencing-server/test/sequence-generation.spec.ts

```diff
@@ -2587,7 +2587,7 @@ describe('sequence generation', () => {
   }, 30000);
 });
 
-describe('expansion', () => {
+describe('expansion regressions', () => {
```

Comment:
We should add a test here or in the e2e-tests that creates an expansion rule, creates an expansion set, and runs expansion.

Reply:
I added a test on line `143` that creates an expansion set to test when there's a name conflict in the command dictionary and model, it's interesting that isn't catching this new error.

Reply:
So I'm testing this a little bit more this morning. It seems to be [this command dictionary](https://github.com/NASA-AMMOS/aerie-ui/blob/develop/e2e-tests/data/command-dictionary.xml) that is causing the error. The [command dictionary in the sequencing server](https://github.com/NASA-AMMOS/aerie/blob/develop/sequencing-server/cdict/command_banananation.xml) does not cause the error.

Reply:
@camargo this test does create an expansion rule, set and run expansions. We do it multiple times in this test.